### DISCUSSION TITLE
听书锁屏后杀后台

### DISCUSSION THREAD

**Discussion by Yaksa0129** [xiaohucode/yidaRule#13] [Q&A]
听书锁屏后杀后台，这个解决不了吗。

**Comment by xiaohucode** [comment]
> 听书锁屏后杀后台，这个解决不了吗。

我没遇到过。可以尝试勾选【后台App刷新】看看是否缓解

**Reply by Yaksa0129** [reply]
最新版我试了下，大概在锁屏后一分钟到两分钟之间杀后台，如果确定没问题那我就重置手机了。
